bazel-haskell-deps.bzl

```diff
@@ -260,6 +260,22 @@ haskell_library(
         urls = ["https://github.com/digital-asset/ghcide/archive/%s.tar.gz" % GHCIDE_REV],
     )
 
+    # The Bazel-provided grpc libs cause issues in GHCi so we get them from Nix on Linux and MacOS.
+    # We keep them in
+    deps = "[\":grpc\", \":libgpr\"]" if is_windows else "[\"@grpc_nix//:grpc_lib\"]"
```

Comment:
Not important, but you can also use `'[":grpc", ...]'` to avoid some escaping.

Reply:
Ah good point, thanks! Changed.